index.html

```diff
@@ -7,6 +7,6 @@
     <title>Document</title>
```

Comment:
- [ ] **Required** Let's add `CSS file link` here to reflect CSS code in html file.

Reply:
Addressed ✔